python/lvmdrp/functions/skyMethod.py

```diff
@@ -1646,7 +1647,7 @@ def quick_sky_subtraction(in_cframe, out_sframe,
     sky_error[skywfibers] = skywsky_error
 
     # write out sky table to ancillary file
-    mjd = cframe._header['SMJD']  # use SMJD to account for exposures taken before the nightly MJD switch
+    mjd = cframe._header['MJD']
```

Comment:
this needs to be the `SMJD`.  Can you revert this commit?

Reply:
Unless this has been fixed and doesn't impact exposures taken before the nightly cutoff?

Reply:
Not sure what happened there.  Saw your earlier commit on Aug 2 and fixed it on this branch back.